### PR TITLE
docs: update the units to make the unit dimensions right

### PR DESCRIPTION
**Pull request by stepashka**
right now it looks like we divide by the amount of hours
https://neon-next-git-stepashka-patch-4-neondatabase.vercel.app/docs/introduction/billing#billing-rates

### CODE REVIEW

**Review by acervantes23 — CHANGES_REQUESTED** [review]
@stepashka the units need to be separated by "-"

**Review by danieltprice — DISMISSED** [review]

File: content/docs/introduction/billing.md [hunk 1]
```diff
@@ -151,23 +151,23 @@ data transfer (GiB) * price per GiB
...
-| AWS            | US East (Ohio)     | Project storage  | $0.000164 | GiB/Hour |
+| AWS            | US East (Ohio)     | Compute time  | $0.10200 | ComputeHour |
+| AWS            | US East (Ohio)     | Project storage  | $0.000164 | GiBHour |
 | AWS            | US East (Ohio)  | Written data  | $0.09600 | GiB |
 | AWS            | US East (Ohio)| Data transfer  | $0.09000 | GiB |
 |                |             |                |       |      |
-| AWS            | US West (Oregon)     | Compute time  | $0.10200 | Compute/Hour |
-| AWS            | US West (Oregon)| Project storage | $0.000164 | GiB/Hour |
+| AWS            | US West (Oregon)     | Compute time  | $0.10200 | ComputeHour |
+| AWS            | US West (Oregon)| Project storage | $0.000164 | GiBHour |
```
Comment (acervantes23) [inline content/docs/introduction/billing.md:160]:
```suggestion
| AWS            | US West (Oregon)| Project storage | $0.000164 | GiB-hour |
```

File: content/docs/introduction/billing.md [hunk 2]
```diff
@@ -151,23 +151,23 @@ data transfer (GiB) * price per GiB
...
 | AWS            | US West (Oregon)     | Written data  | $0.09600 | GiB |
 | AWS            | US West (Oregon)  | Data transfer  | $0.09000 | GiB |
 |                |             |                |       |      |
-| AWS            | Europe (Frankfurt)| Compute time| $0.11800 | Compute/Hour |
-| AWS            | Europe (Frankfurt)| Project storage  | $0.00018 | GiB/Hour |
+| AWS            | Europe (Frankfurt)| Compute time| $0.11800 | ComputeHour |
```
Comment (acervantes23) [inline content/docs/introduction/billing.md:164]:
```suggestion
| AWS            | Europe (Frankfurt)| Compute time| $0.11800 | Compute-hour |
```

File: content/docs/introduction/billing.md [hunk 3]
```diff
@@ -151,23 +151,23 @@ data transfer (GiB) * price per GiB
...
+| AWS            | Europe (Frankfurt)| Project storage  | $0.00018 | GiBHour |
```
Comment (acervantes23) [inline content/docs/introduction/billing.md:165]:
```suggestion
| AWS            | Europe (Frankfurt)| Project storage  | $0.00018 | GiB-hour |
```

File: content/docs/introduction/billing.md [hunk 4]
```diff
@@ -151,23 +151,23 @@ data transfer (GiB) * price per GiB
...
 | AWS            | Europe (Frankfurt)     | Written data    | $0.09600 | GiB |
 | AWS            | Europe (Frankfurt)     | Data transfer    | $0.09000 | GiB |
 |                |             |                |       |      |
-| AWS            | Asia Pacific (Singapore)| Compute time    | $0.12100 | Compute/Hour |
-| AWS            | Asia Pacific (Singapore)  | Project storage    | $0.00018 | GiB/Hour |
+| AWS            | Asia Pacific (Singapore)| Compute time    | $0.12100 | ComputeHour |
```
Comment (acervantes23) [inline content/docs/introduction/billing.md:169]:
```suggestion
| AWS            | Asia Pacific (Singapore)| Compute time    | $0.12100 | Compute-hour |
```

File: content/docs/introduction/billing.md [hunk 5]
```diff
@@ -151,23 +151,23 @@ data transfer (GiB) * price per GiB
...
+| AWS            | Asia Pacific (Singapore)  | Project storage    | $0.00018 | GiBHour |
```
Comment (acervantes23) [inline content/docs/introduction/billing.md:170]:
```suggestion
| AWS            | Asia Pacific (Singapore)  | Project storage    | $0.00018 | GiB-hour |
```

**Review by acervantes23 — CHANGES_REQUESTED** [review]

File: content/docs/introduction/billing.md [hunk 6]
```diff
@@ -151,23 +151,23 @@ data transfer (GiB) * price per GiB
 
 | Cloud provider | Region      | Billing metric | Price | Unit |
 |:---------------|:-----------|:--------------|:------|:-----|
-| AWS            | US East (Ohio)     | Compute time  | $0.10200 | Compute/Hour |
-| AWS            | US East (Ohio)     | Project storage  | $0.000164 | GiB/Hour |
+| AWS            | US East (Ohio)     | Compute time  | $0.10200 | ComputeHour |
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/introduction/billing.md]: fixed

**Review by danieltprice — APPROVED** [review]

**Review by acervantes23 — APPROVED** [review]

**Pull request merged**